openscm/adapters/__init__.py

```diff
@@ -68,6 +68,7 @@ def initialize_model_input(self) -> None:
         Called before the adapter is used in any way and at most once before a call to
         :func:`run` or :func:`step`.
         """
+        # I don't know why this isn't picked up by code coverage...
```

Comment:
Do we really want to keep these comments? ;)

Reply:
Nope just making sure you've seen it